<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dAccesoConfigurarTerminal</class>
 <widget class="QDialog" name="dAccesoConfigurarTerminal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>135</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>135</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>135</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Acceso para programadores</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Ticket/Version 1/img/logotipo1.png</normaloff>../Ticket/Version 1/img/logotipo1.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton_Acceder">
   <property name="geometry">
    <rect>
     <x>367</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Acceder</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="palabraMagica">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>400</width>
     <height>32</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>Configurar</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Corte</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
